inference-engine/ie_bridges/python/wheel/meta/openvino-dev.requirements.txt

```diff
@@ -0,0 +1,22 @@
+defusedxml>=0.5.0
+scipy==1.2.1
+jstyleson==0.0.2
+numpy==1.16.3
+addict==2.2.1
+pandas==0.24.2
+hyperopt==0.1.2
+networkx==2.2
+tqdm==4.31.1
+py-cpuinfo==4.0.0
+PyYAML
+pillow>=2.6.1
+scikit-image
+scikit-learn
+yamlloader
+shapely
+nibabel
+pydicom
+sentencepiece
+tokenizers
+editdistance
```

Comment:
which principies has this requirements list? Should it includes all requirements for the tools or not?
I am not sure that all AC requirements here (missed nltk, parasail, fast-ctc-decode, rawpy)

Reply:
I've taken those requirements from your setup.py. It should include all mandatory requiremens  for the tool
If we missed something, please define them here